src/test/ui/consts/miri_unleashed/mutable_const.stderr

```diff
@@ -11,7 +11,7 @@ LL | / const MUTATING_BEHIND_RAW: () = {
 LL | |     // Test that `MUTABLE_BEHIND_RAW` is actually immutable, by doing this at const time.
 LL | |     unsafe {
 LL | |         *MUTABLE_BEHIND_RAW = 99
-   | |         ^^^^^^^^^^^^^^^^^^^^^^^^ writing to alloc1 which is read-only
+   | |         ^^^^^^^^^^^^^^^^^^^^^^^^ modifying a static's initial value from another static's initializer
```

Comment:
This changed because we now check "is it mutating" (and error because we cannot mutate any global memory) before we check "is the target memory read-only".

Maybe I should try getting the original error back...